Sources/Config.swift

```diff
@@ -2,14 +2,14 @@ import UIKit
 
 public struct Config {
 
-  public static var topLineColor: UIColor = UIColor.darkGray.withAlphaComponent(0.8)
+  public static var topLineColor: UIColor = UIColor.darkGray.withAlphaComponent(0.5)
 
   public struct ArrowButton {
 
     public struct Text {
       public static var color: UIColor = UIColor.white
       public static var selectedColor: UIColor = UIColor.gray
-      public static var font: UIFont = UIFont.boldSystemFont(ofSize: 18)
+      public static var font: UIFont = UIFont(name: "Circe", size: 14) ?? UIFont.boldSystemFont(ofSize: 14)
```

Comment:
Why do you use this `Circe` font ?

Reply:
I used this font in my project and forgot to remove it